config/webpack/base.js

```diff
@@ -13,6 +13,7 @@ var fontsConfig = require("./partial/fonts");
 var imagesConfig = require("./partial/images");
 var statsConfig = require("./partial/stats");
 var isomorphicConfig = require("./partial/isomorphic");
+var manifestConfig = require('./partial/manifest');
```

Comment:
Probably rename this to `pwa` or something, or separate the manfiest stuff from the service worker stuff.